content/academy/pro_apify_scraper_development/actor_building.md

```diff
@@ -0,0 +1,518 @@
+---
+title: Actor building
+description: Build a pro Apify actor. With all of the code provided right within the lesson, you can learn advanced scraping techniques by coding along!
+menuWeight: 6.2
+paths:
+    - pro-apify-scraper-development/actor-building
+---
+
+# [](#setup) Setup
+
+The Apify CLI makes it extremely easy for us to set up a project in the SDK and hit the ground running. Navigate to the directory you'd like your project's folder to live, then open up a terminal instance and run the following command:
+
+```shell
+apify create demo-actor
+```
+
+> You don't have to call it **demo-actor**, but that's what we'll be calling it within this tutorial.
+
+Once you run this command, you'll get prompted into a menu which you can navigate using your arrow keys. Each of these options will generate different boilerplate code when selected. We're going to work with CheerioCrawler today, so we'll select that one, then press <p><kbd>Enter</kbd><p>.
```

Comment:
Looks like the `<p>` tag messes up the formatting - pls remove

<img width="280" alt="Screenshot 2022-04-13 at 10 25 25" src="https://user-images.githubusercontent.com/38246758/163133459-e447e39b-246c-417b-8ee3-ba4d29559de5.png">